channels/serializers.py

```diff
@@ -900,4 +900,4 @@ def get_reasons(self, instance):
         Returns:
             list of str: list of reasons a post/comment has been reported for
         """
-        return sorted(list(set([report[0] for report in instance.user_reports + instance.mod_reports])))
+        return sorted(list({report[0] for report in instance.user_reports + instance.mod_reports}))
```

Comment:
You can also remove the `list` at the beginning since `sorted` will accept a set as input

Reply:
👍